composer.json

```diff
@@ -12,17 +12,17 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": "^7.2||^8.0",
         "symfony/framework-bundle": "^4.4 || ^5.0",
         "symfony/http-client": "^4.4 || ^5.0",
         "symfony/http-client-contracts": "^1.0|^2.0",
         "nyholm/psr7": "^1.0",
-        "m4tthumphrey/php-gitlab-api": "^10.0"
+        "m4tthumphrey/php-gitlab-api": "^10.3"
     },
     "require-dev" : {
         "phpunit/phpunit": "^8.0",
         "guzzlehttp/psr7": "^1.5.2",
-        "php-http/guzzle6-adapter": "^2.0.1",
+        "php-http/guzzle7-adapter": "^0.1.1",
```

Comment:
This line and the one above should be deleted, and replaced with `guzzlehttp/guzzle` at `^7.2`.

Reply:
You replaced this line and the one below, instead of the one above. :P